tests/tensorflow/r2.12/dlrm.libsonnet

```diff
@@ -0,0 +1,263 @@
+            warmup_steps: 8000,
+          },
+        },
+      },
+
+    },
+    command: [
+      'python3',
+      'official/recommendation/ranking/train.py',
+      '--params_override=%s' % (std.manifestYamlDoc(self.paramsOverride) + '\n'),
+      '--model_dir=$(MODEL_DIR)',
+    ],
+  },
+  local functional = common.Functional {
+    command+: [
+      '--mode=train',
+    ],
+    paramsOverride+: {
+      trainer+: {
+        train_steps: 10000,
+      },
+    },
+  },
+  local convergence = common.Convergence {
+    local config = self,
+
+    command+: [
+      '--mode=train_and_eval',
+    ],
+    paramsOverride+: {
+      trainer+: {
+        train_steps: 256054,
+      },
+    },
+  },
+
+  local gpu_common = {
+    local config = self,
+
+    paramsOverride+:: {
+      runtime+: {
+        distribution_strategy: 'mirrored',
+        num_gpus: config.accelerator.count,
+      },
+      task+: {
+        model+: {
+          bottom_mlp: [512, 256, 8],
+          embedding_dim: 8,
+        },
+      },
+    },
+  },
+
+  local cross_interaction = {
+    modelName: 'ranking-dcn',
+    local config = self,
+
+    paramsOverride+:: {
+      task+: {
+        model+: {
+          interaction: 'cross',
+        },
+      },
+    },
+  },
+
+  local v100 = gpu_common {
+    accelerator: gpus.teslaV100,
+  },
+  local v100x4 = v100 {
+    accelerator: gpus.teslaV100 { count: 4 },
+  },
+
+  local tpu_common = {
+    paramsOverride+:: {
+      runtime+: {
+        distribution_strategy: 'tpu',
+        tpu: '$(KUBE_GOOGLE_CLOUD_TPU_ENDPOINTS)',
+      },
+    },
+  },
+
+  local v2_8 = tpu_common {
+    accelerator: tpus.v2_8,
+    paramsOverride+:: {
+      task+: {
+        model+: {
+          bottom_mlp: [512, 256, 16],
+          embedding_dim: 16,
+        },
+      },
+    },
+  },
+
+  local v2_32 = tpu_common {
+    accelerator: tpus.v2_32,
+    paramsOverride+:: {
+      task+: {
+        model+: {
+          bottom_mlp: [512, 256, 64],
+          embedding_dim: 64,
+        },
+      },
+    },
+  },
+
+  local v3_8 = tpu_common {
+    accelerator: tpus.v3_8,
+    paramsOverride+:: {
+      task+: {
+        model+: {
+          bottom_mlp: [512, 256, 32],
+          embedding_dim: 32,
+        },
+      },
+    },
+  },
+
+  local v3_32 = tpu_common {
+    accelerator: tpus.v3_32,
+    paramsOverride+:: {
+      task+: {
+        model+: {
+          bottom_mlp: [512, 256, 128],
+          embedding_dim: 128,
+        },
+      },
+    },
+  },
+
+  local v4_8 = tpu_common {
+    accelerator: tpus.v4_8,
+    paramsOverride+:: {
+      task+: {
+        model+: {
+          bottom_mlp: [512, 256, 64],
+          embedding_dim: 64,
+        },
+      },
+    },
+  },
+  local v4_32 = tpu_common {
+    accelerator: tpus.v4_32,
+    paramsOverride+:: {
+      task+: {
+        model+: {
+          bottom_mlp: [512, 256, 128],
+          embedding_dim: 128,
+        },
+      },
+    },
+  },
+  local tpuVm = common.tpuVm,
```

Comment:
tpu.Vm is not there in common.libsonnet.
Reference: https://github.com/gkroiz/ml-testing-accelerators/blob/5962332e531c7858cc68c9463668434b9c45ef38/tests/tensorflow/r2.11/common.libsonnet#L30